### PR TITLE
Modifies an example, adds advanced exercises and fixes typos

### PR DESCRIPTION
**Pull request by JohannesNE**
Modified the storms example to give a more compact dataframe as result.
Also added an output example to exercise 4 and added two suggestions for advanced versions of other exercises.

### CODE REVIEW

File: content/material/2020-01-17-intermediate-dplyr.Rmd [hunk 1]
```diff
@@ -1,21 +1,21 @@
...
-- "Johannes Enevoldsen"
-- "Sara Tomiolo"
-- "Luke Johnston"
+author:
+- Johannes Enevoldsen
+- Sara Tomiolo
+- Luke Johnston
 date: "1/17/2020"
-description: >
+description: |
```
Comment (lwjohnst86) [inline content/material/2020-01-17-intermediate-dplyr.Rmd:8]: Either > or | works, just for next time.

**Review by lwjohnst86 — APPROVED** [review]
Looks good! Merging in :)

**Comment by lwjohnst86** [comment]
See you tomorrow! We'll be there maybe 10 min before hand. :)

**Comment by JohsEnevoldsen** [comment]
Perfect. See you.

**Pull request merged**